model/resource/android.yaml

```diff
@@ -0,0 +1,14 @@
+groups:
+  - id: android
+    prefix: android
+    type: resource
+    brief: >
+      The android platform on which the android application is running.
+    attributes:
+      - id: os.api.level
```

Comment:
I wonder if we really need the `os` namespace here. Isn't `android` already enough to indicate it's the platform/OS? I didn't give it much thought, it is just something that I notice now while reviewing. Did the SIG discuss it in any way? 

Additionally, do you think there will be more attributes under `api` to warrant if being it's own namespace?

Reply:
> I wonder if we really need the os namespace here. Isn't android already enough to indicate it's the platform/OS? I didn't give it much thought, it is just something that I notice now while reviewing. Did the SIG discuss it in any way?

This came up in the SIG, we discussed adding `os` since there could be other android related attrs that might refer to the app instead of the os, not necessarily related to this particular attr, though having everything os-related under `os` sounds cleaner and could help avoid misunderstandings.

> Additionally, do you think there will be more attributes under api to warrant if being it's own namespace?

This is a good point we didn't discuss in the last SIG, I can't think of any other attrs needed under `api` atm, so I'm ok to rename it to something else such as `api_level` for example.

Reply:
Thanks a lot for the reviews and feedback! I've updated api.level to api_level as we do not foresee any other attributes under os.api namespace. :)